web/src/components/CardHeader.vue

```diff
@@ -0,0 +1,54 @@
+<template>
+  <v-row>
+    <h2>{{ title }}</h2>
+    <v-spacer />
+    <v-btn
+      icon
+      large
+      class="ml-1"
+      @click="toggleFavorite"
+    >
+      <v-icon>
+        {{ star }}
+      </v-icon>
+    </v-btn>
+    <v-btn
+      icon
+      large
+      class="ml-1"
+      @click="contact"
+    >
+      <v-icon>email</v-icon>
+    </v-btn>
+  </v-row>
+</template>
+
+<script>
+export default {
+  props: {
+    title: {
+      type: String,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      favorited: false,
+    };
+  },
+  computed: {
+    star() {
+      return this.favorited ? 'star' : 'star_border';
+    },
+  },
+  methods: {
+    toggleFavorite() {
+      this.favorited = !this.favorited;
+      // TODO: find a way to save this
+    },
```

Comment:
Is this a new feature to favorite speakers? I think a feature like this would require more discussion (like having user profiles, etc) that is not originally in our MVP scope. Is it okay to remove it for now and add it in the future when we decide in what direction to expand the MVP to?

Reply:
Sure thing, it's easy to hide for now.